lib/ansible/modules/cloud/vmware/vmware_datastore_cluster.py

```diff
@@ -0,0 +1,159 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright (c) 2018, Ansible Project
+# Copyright (c) 2018, Abhijeet Kasurde <akasurde@redhat.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+
+DOCUMENTATION = '''
+---
+module: vmware_datastore_cluster
+short_description: Manage VMware vSphere datastore clusters
+description:
+    - This module can be used to add and delete datastore cluster in given VMware environment.
+    - All parameters and VMware object values are case sensitive.
+version_added: 2.6
+author:
+-  Abhijeet Kasurde (@akasurde)
+notes:
+    - Tested on vSphere 6.0, 6.5
+requirements:
+    - "python >= 2.6"
+    - PyVmomi
+options:
+    datacenter_name:
+      description:
+      - The name of the datacenter.
+      required: True
+    datastore_cluster_name:
+      description:
+      - The name of the datastore cluster.
+      required: True
+    state:
+      description:
+      - If the datacenter should be present or absent
```

Comment:
I assume `If the datastore cluster should be present or absent`

Reply:
OK.

Reply:
OK.